src/autopas/containers/octree/traversals/OTTraversalInterface.h

```diff
@@ -18,14 +18,66 @@ namespace autopas {
 /**
  * This interface exists to provide a row interface for octree to add its cells.
  */
-template <typename ParticleCell>
+template <typename Particle, typename ParticleCell>
```

Comment:
I think this template parameter should be unnecessary as you should be able to get the Particle type from the cell